modules/nf-core/gatk4/postprocessgermlinecnvcalls/Dockerfile

```diff
@@ -0,0 +1,102 @@
+    apt update && \
+    apt-key list && \
+    apt-get -y clean  && \
+    apt-get -y autoclean  && \
+    apt-get -y autoremove && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /gatk
+
+RUN chmod -R a+rw /gatk
+# Location of the unzipped gatk bundle files
+COPY --from=gradleBuild /gatk/unzippedJar .
+
+#Setup linked jars that may be needed for running gatk
+RUN ln -s $( find /gatk -name "gatk*local.jar" ) gatk.jar
+RUN ln -s $( find /gatk -name "gatk*local.jar" ) /root/gatk.jar
+RUN ln -s $( find /gatk -name "gatk*spark.jar" ) gatk-spark.jar
+
+WORKDIR /root
+
+# Make sure we can see a help message
+RUN java -jar gatk.jar -h
+RUN mkdir /gatkCloneMountPoint
+RUN mkdir /jars
+RUN mkdir .gradle
+
+WORKDIR /gatk
+
+# Create a simple unit test runner
+ENV CI true
+# "export GATK_DOCKER_CONTAINER=true" is used to allow tests to determine when the're running on the docker
+# (some negative python tests use this to throw skip exceptions). See GATKBaseTest::isGATKDockerContainer.
+RUN echo "source activate gatk" > /root/run_unit_tests.sh && \
+    echo "export GATK_DOCKER_CONTAINER=true" >> /root/run_unit_tests.sh && \
+    echo "export TEST_JAR=\$( find /jars -name \"gatk*test.jar\" )" >> /root/run_unit_tests.sh && \
+    echo "export TEST_DEPENDENCY_JAR=\$( find /jars -name \"gatk*testDependencies.jar\" )" >> /root/run_unit_tests.sh && \
+    echo "export GATK_JAR=$( find /gatk -name "gatk*local.jar" )" >> /root/run_unit_tests.sh && \
+    echo "export GATK_LAUNCH_SCRIPT=/gatk/gatk" >> /root/run_unit_tests.sh && \
+    echo "mkdir /gatk/srcdir" >> /root/run_unit_tests.sh && \
+    echo "cp -rp /gatkCloneMountPoint/src/main/java/* /gatk/srcdir" >> /root/run_unit_tests.sh && \
+    echo "export SOURCE_DIR=/gatk/srcdir" >> /root/run_unit_tests.sh && \
+    echo "export GRADLE_OPTS=\"-Xmx1024m -Dorg.gradle.daemon=false --add-opens java.prefs/java.util.prefs=ALL-UNNAMED\"" >> /root/run_unit_tests.sh && \
+    echo "export CP_DIR=/gatk/testClasses" >> /root/run_unit_tests.sh && \
+    echo "ln -s /gatkCloneMountPoint/src/ /gatkCloneMountPoint/scripts/docker/src" >> /root/run_unit_tests.sh && \
+    echo "ln -s /gatkCloneMountPoint/build/ /gatkCloneMountPoint/scripts/docker/build" >> /root/run_unit_tests.sh && \
+    echo "cd /gatk/ && /gatkCloneMountPoint/gradlew -Dfile.encoding=UTF-8 -b /gatkCloneMountPoint/dockertest.gradle testOnPackagedReleaseJar jacocoTestReportOnPackagedReleaseJar -a -p /gatkCloneMountPoint" >> /root/run_unit_tests.sh
+
+WORKDIR /root
+RUN cp -r /root/run_unit_tests.sh /gatk
+RUN cp -r gatk.jar /gatk
+ENV CLASSPATH /gatk/gatk.jar:$CLASSPATH
+
+# Start GATK Python environment
+
+WORKDIR /gatk
+ENV PATH $CONDA_PATH/envs/gatk/bin:$CONDA_PATH/bin:$PATH
+RUN conda env create -n gatk -f /gatk/gatkcondaenv.yml && \
+    echo "source activate gatk" >> /gatk/gatkenv.rc && \
+    echo "source /gatk/gatk-completion.sh" >> /gatk/gatkenv.rc && \
+    conda clean -afy && \
+    rm -rf /root/.cache/pip
```

Comment:
@matthdsm Are these the lines?

Reply:
Nah, I think its a pip install somewhere at the bottom of the dockerfile. 
Can’t look it right now, on mobile

Reply:
It’s this line here. 
https://github.com/broadinstitute/gatk/blob/cfd4d87ec29ac45a68f13a37f30101f326546b7d/scripts/gatkcondaenv.yml.template#L66
Afaik, this should install the custom scripts, but for some reason, these aren’t detected when the conda recipe is used